CONTRIBUTING.md

```diff
@@ -43,6 +43,17 @@ Your shell prompt will be augmented by the `(venv)` prefix, e.g.
 (venv) mylaptop:~/fcc/fcc-tutorials
 ```
 
```

Comment:
Should we add that for CentOS7 (lxplus) it is required to have a more recent Python than the one provided by the system and that a way to get it is to enable Key4hep or FCCSW?